ztp/kube-compare-reference/optional/ptp-config/PtpConfigDualFollower.yaml

```diff
@@ -0,0 +1,138 @@
+# In this example, 2 ports of a single NIC are used in a active/standby configuration to synchronize to a 
+# remote grandmaster via interfaces $iface1 and $iface1
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: ordinary-clock-1
```

Comment:
Maybe a name like `dual-ordinary-clock` would be more descriptive?

Reply:
Fixed